suites/squid/common/regression/rados/rados_3-AZ-Cluster.yaml

```diff
@@ -0,0 +1,84 @@
+# 3 AZ cluster deployment tests
+# Conf: conf/squid/common/9node-1client-availability-zone.yaml
+# Deployment: suites/squid/common/regression/AZ_cluster_deploy_and_configure.yaml
+tests:
+
+  - test:
+      name: Enable logging to file
+      module: rados_prep.py
+      config:
+        log_to_file: true
+      desc: Change config options to enable logging to file
+
+  - test:
+      name: Enable Stretch mode on pools
+      module: pool_tests.py
+      polarion-id: CEPH-83609796
+      config:
+        enable_3AZ_stretch_pools:
+          pool_name: test-pool-11
+          stretch_bucket: datacenter
+          rule_name: 3az_rule
+      desc: Enable 3 AZ stretch mode on all pools of the cluster
+
+  - test:
```

Comment:
3AZ test suite includes Upgrade workflow. Should this test be included here?

Reply:
This is going to be a separate deployment altogether, so should be okay